salt/utils/http.py

```diff
@@ -532,6 +533,7 @@ def query(url,
                 allow_nonstandard_methods=True,
                 streaming_callback=streaming_callback,
                 header_callback=header_callback,
+                connect_timeout=connect_timeout,
```

Comment:
Can you move this to the bottom of the kwargs list? We don't want to cause an API change for people or other Salt functions using `query` who _might_ be calling this with args/kwargs in a certain order.

Reply:
This is the call to fetch() - http://www.tornadoweb.org/en/stable/httpclient.html#tornado.httpclient.HTTPRequest (you need to expand the minimized lines to see this, otherwise this looks like query() parameter list), and there connect_timeout comes before request_timeout.

Reply:
Oh! You're right. My apologies.